### PR TITLE
Submit Gluu 1 click kubernetes app setup

### PR DESCRIPTION
**Pull request by moabu**
## BACKGROUND
* Submit Gluu Cloud Native Edition one click Kubernetes app setup

-----------------------------------------------------------------------

## Changes
* Added `deploy.sh` and `README.md`

-----------------------------------------------------------------------

## Checklist
- [x] review the [contributing doc](https://github.com/digitalocean/marketplace-kubernetes/blob/master/CONTRIBUTING.md) with steps for both adding or updating your application (if applicable)
------------------------------------------------------------------------

Reviewer: @marketplace-eng

### CODE REVIEW

**Comment by moabu** [comment]
@paulanunda there is, but its a bit more complicated as we are trying to show all Gluu services in an easy install. It will require work with the bash script to replace python3 work. If thats a big issue we might switch to applying the installer as a job.

**Comment by paulanunda** [comment]
@moabu A job manifest would be preferable.

**Comment by moabu** [comment]
@paulanunda I have switched the files to use two job manifests. I hope this works with you guys.

**Comment by paulanunda** [comment]
@moabu Thanks, I'll review today.

File: stacks/gluu/deploy-local.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+set -e
+
+ROOT_DIR=$(git rev-parse --show-toplevel)
+
+# Create clusterrolebbinding for default sa
+kubectl create clusterrolebinding gluu-cluster-adminn \
```
Comment (paulanunda) [inline stacks/gluu/deploy-local.sh:8]: Does `gluu-cluster-adminn` have a typo?

**Pull request merged**